.eslintignore

```diff
@@ -1,4 +1,5 @@
 .cpanm/**/*.js
+flow-typed/npm/@sentry/*.js
```

Comment:
This contained Flow syntax that eslint didn't like (mostly related to semicolons and lack of indexer key names). I've only ignored this specific libdef because other ones we've written ourselves, and I'd still like those checked.